CMakeLists.txt

```
@@ -12,7 +12,7 @@ if("${Boost_VERSION}" VERSION_LESS "${VCLIBS_MIN_BOOST_VERSION}")
     message(FATAL_ERROR "Detected Boost version is too old (older than ${VCLIBS_MIN_BOOST_VERSION}).")
 endif()
 
-set(VCLIBS_SUFFIX "_oss")
+set(VCLIBS_SUFFIX "_oss" CACHE STRING "Additional suffix appended to build dlls to not conflict with the one used by the OS or other applications.")

```

* "appended" is redundant with "suffix" and partially redundant with "Additional"
* acronyms like "DLL" are typically capitalized
* there are multiple DLLs we are avoiding name conflicts with, not just one
* the user of said DLLs isn't really important, so much as that they are the copies shipped with VS/in the redist
```suggestion
set(VCLIBS_SUFFIX "_oss" CACHE STRING "suffix for built DLL names to avoid conflicts with distributed DLLs")
```